marc_to_solr/lib/process_holdings_helpers.rb

```diff
@@ -59,7 +59,10 @@ def select_permanent_location_876(group_876_fields, field_852)
   # Select 876 fields (items) with current location. 876 location is NOT equal to the 852 permanent location.
   def select_temporary_location_876(group_876_fields, field_852)
     return [] if /^scsb/.match?(field_852['b'])
-    group_876_fields.select { |field_876| current_location_code(field_876) != permanent_location_code(field_852) }
+    group_876_fields.select do |field_876| 
+      current_location = current_location_code(field_876) 
+      current_location != permanent_location_code(field_852) && current_location != 'RES_SHARE$IN_RS_REQ'
+    end
```

Comment:
@carolyncole the current_location is = 'RES_SHARE$IN_RS_REQ'. 
We want when an item (876) is in this location to index it as a permanent location and not to index it as a temporary location.

Reply:
@christinach yes that is what this code does.  Select it if the temporary location is NOT the primary location and the Temporary Location is NOT RES_SHARE$IN_RS_REQ

Reply:
Just to clarify, _current location_ is the location of any item in 876. _Temporary location_ is the logic of having an item in 876 with a different location of the equivalent 852 location.

Reply:
@christinach I added a comment to the code.  Does that help?

Reply:
@carolyncole we want to index this item as a permanent holding. We dont want to ignore it.

Reply:
We ignore it from indexing it as temporary but we account it and index it as permanent.

Reply:
@christinach this change is just removing these items from the temporary select.  I'm totally unsure what you are getting at here.  Can we zoom?

Reply:
yes, I understand that you remove it from the temporary. but it should be added in the permanent holding and in the end have a holdings_1display like the one I put it the spec. I asked @mzelesky to confirm the content of the holdings_1display.